src/main/java/gift/domain/wishlist/service/WishService.java

```diff
@@ -44,7 +45,16 @@ public Page<WishResponse> getWishesByMember(Member member, int pageNo, int pageS
 
     @Transactional
     public WishResponse createWish(WishRequest wishRequest) {
-        Wish wish = DtoToEntity(wishRequest);
+        Member member = memberRepository.findById(wishRequest.getMemberId())
+            .orElseThrow(() -> new MemberNotFoundException("해당 유저가 존재하지 않습니다."));
+        Product product = productRepository.findById(wishRequest.getProductId())
+            .orElseThrow(() -> new ProductNotFoundException("해당 상품이 존재하지 않습니다."));
+
+        if (wishRepository.findByProductAndMember(product, member).isPresent()) {
+            throw new WishDuplicateException("중복된 위시리스트 입니다.");
+        }
```

Comment:
JPA의 도움을 받아 이렇게도 사용할 수 있습니다 ㅎㅎ
```suggestion
        if (wishRepository.existsByProductAndMember(product, member)) {
            throw new WishDuplicateException("중복된 위시리스트 입니다.");
        }
```